.github/workflows/test-python-hasi.yml

```diff
@@ -0,0 +1,41 @@
+name: Build, test, package
+
+on: [push,pull_request]
+
+env:
+  ITKBoneEnhancement-git-tag: v0.4.1
+
+jobs:
+  test-python-hasi-package:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        include:
+          - os: ubuntu-18.04
+          - os: windows-2019
+          - os: macos-10.15
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Move to hasi module directory
```

Comment:
Could this command be merged with the last one (`pytest .`)? If not, maybe rename to `change directory to HASI source`?

Reply:
Good point. Added `cd` command to final `pytest` step.